packages/machines/dialog/src/dialog.types.ts

```diff
@@ -69,6 +69,11 @@ type PublicContext = DirectionProperty & {
    * @default "dialog"
    */
   role: "dialog" | "alertdialog"
+  /**
+   * Whether to open or close the dialog on setup
+   * @default false
+   */
+  isOpen?: boolean
```

Comment:
I feel `defaultOpen` is better here since it's not controlled
```suggestion
  isOpen?: boolean
```

Reply:
I agree that it's probably better to have "default" somewhere in it as `isOpen` can be misinterpreted as a controlled variable.

Reply:
The notion of controller and uncontrolled is something we need to be cautious about as it’s a concept that’s generalized by React. 

I think we need to broaden our scope beyond React and see as if we’re building DOM primitives. Solid.js for example, doesn’t have the notion of controlled-uncontrolled since it compiles directly to the DOM (no VDOM)

Native APIs use naming like value, open, checked, etc to denote the initial state of the element. I recommend we stick with this approach.

Reply:
Makes sense, thanks for the explanation.

Anything else I can do to move the PR forward?